pkg/compiler/const_file.go

```diff
@@ -90,6 +90,8 @@ func (cf *ConstFile) Serialize() []byte {
 	for arch := range cf.arches {
 		arches = append(arches, arch)
 	}
+	arches = append(arches, "riscv64")
```

Comment:
This looks wrong. Some leftovers?

Reply:
Fixed in https://github.com/google/syzkaller/pull/3253/commits/b6b8f5bdad6961fa580ed9fd254a8a8e54ef3a60.